libtenzir/builtins/operators/from_load_read.cpp

```diff
@@ -57,7 +57,7 @@ class load_operator final : public crtp_operator<load_operator> {
     // TODO: Fuse this check with crtp_operator::instantiate()
     return caf::make_error(ec::type_clash,
                            fmt::format("'{}' does not accept {} as input",
-                                       to_string(), operator_type_name(input)));
+                                       name(), operator_type_name(input)));
```

Comment:
See above.

Reply:
Nice catch. I changed the remaining instances as well and double-checked via `git-grep` that I caught 'em all now.